src/main/java/practice/StreamPractice.java

```diff
@@ -34,8 +48,12 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * <p>
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
+
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(people -> people.getSex().equals(Person.Sex.MAN)
```

Comment:
```suggestion
                .filter(people -> people.getSex() == Person.Sex.MAN
```
you can compare enums in such a way